specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,73 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for humans and systems. The FOCUS specification does not require a specific level of precision for numeric values. The level of precision required for a given column is determined by the provider and should be part of a data definition published by the provider.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST follow the formatting requirements listed below. Provider generated numeric value capturing columns SHOULD adopt the same format requirements over time.
+
+## Attribute ID
+
+NumericFormat
+
+## Attribute name
+
+Numeric Format
+
+## Description
+
+Rules and formatting requirements for numeric columns appearing in billing data.
+
+## Requirements
+
+* Columns with a Numeric value format MUST contain a single numeric value.
+* Numeric values MUST be expressed as integer or decimal values. Fractional notation MUST NOT be used.
+* Numeric values MUST NOT be expressed with mathematical symbols, operators, or exponent values.
+* Numeric values MUST NOT contain qualifiers or additional characters (e.g., currency symbols, units of measure, etc.).
+* Numeric values MUST NOT contain commas or punctuation marks except for a single decimal point if required to express a decimal value.
```

Comment:
Should we be explicit about the "decimal point" being a period given regional variations?
```suggestion
* Numeric values MUST NOT contain commas or punctuation marks except for a single decimal point (`.`) if required to express a decimal value.
```